src/anchos.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+const Web3 = require('web3');
+const Mosaic = require('@openstfoundation/mosaic.js');
+const program = require('commander');
+const fs = require('fs');
+
+const StateRootAnchorService = require('./state_root_anchor_service.js');
+
+program
+  .version('0.1.0', '-v, --version')
```

Comment:
you can read the version number directly from package.json:
```js
const { version } = require('../package.json');
```
See https://github.com/OpenSTFoundation/mosaic.js/pull/45/files#diff-eaf8d0f597a8890491d171a0df75da67

Reply:
Nice. Done.